jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsConstants.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2019, Red Hat Inc., and individual contributors as indicated
+ * by the @authors tag. See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.jboss.as.jaxrs;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.jboss.as.controller.SimpleAttributeDefinition;
+
+public class JaxrsConstants {
+
+    public static final String JAXRS_2_0_REQUEST_MATCHING = "jaxrs-2-0-request-matching";
+    public static final String RESTEASY_ADD_CHARSET = "resteasy-add-charset";
+    public static final String RESTEASY_BUFFER_EXCEPTION_ENTITY = "resteasy-buffer-exception-entity";
+    public static final String RESTEASY_DISABLE_HTML_SANITIZER = "resteasy-disable-html-sanitizer";
+    public static final String RESTEASY_DISABLE_PROVIDERS = "resteasy-disable-providers";
+    public static final String RESTEASY_DOCUMENT_EXPAND_ENTITY_REFERENCES = "resteasy-document-expand-entity-references";
+    public static final String RESTEASY_DOCUMENT_SECURE_DISABLE_DTDS = "resteasy-document-secure-disableDTDs";
+    public static final String RESTEASY_DOCUMENT_SECURE_PROCESSING_FEATURE = "resteasy-document-secure-processing-feature";
+    public static final String RESTEASY_GZIP_MAX_INPUT = "resteasy-gzip-max-input";
+    public static final String RESTEASY_JNDI_RESOURCES = "resteasy-jndi-resources";
+    public static final String RESTEASY_LANGUAGE_MAPPINGS = "resteasy-language-mappings";
+    public static final String RESTEASY_MEDIA_TYPE_MAPPINGS = "resteasy-media-type-mappings";
+    public static final String RESTEASY_MEDIA_TYPE_PARAM_MAPPING = "resteasy-media-type-param-mapping";
+    public static final String RESTEASY_PREFERJACKSONOVERJSONB = "resteasy-preferJacksonOverJsonB";
```

Comment:
The name of this sticks out a bit - is it possible to make it follow the normal conventions? e.g. resteasy-prefer-jackson-jsonb

Reply:
Yeah, I agree. I can change RESTEASY_PREFERJACKSONOVERJSONB in wildfly-jaxrs, but, unfortunately, "resteasy.preferJacksonOverJsonB" is already defined in RESTEasy.

Done.